elements/rh-footer/docs/10-style.md

```diff
@@ -47,6 +47,8 @@ theme.
 
 ### Website-specific footer 
```

Comment:
Some of these headings have nothing but token tables below them. We might want to move those headings into the shortcode as well.

Reply:
Example: https://deploy-preview-1077--red-hat-design-system.netlify.app/elements/footer/style/#website-specific-footer

These headers only contained token tables